R/scregclust.R

```diff
@@ -1066,6 +1083,10 @@ scregclust <- function(expression,
 
   # Pre-compute cross-correlation matrices
   cross_corr1 <- fast_cor(z1_target_centered, z1_reg_scaled)
+
+  # Store a copy of abs correlation matrix for quick_mode selection of noise targets
+  abs_corr_matrix <- abs(cross_corr1)
```

Comment:
Only store if quick mode is actually active to save memory.

Reply:
Done!